roles/kubernetes/node/tasks/pre_upgrade.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: "Pre-upgrade | copy /var/lib/cni from kubelet"
+  command: docker cp kubelet:/var/lib/cni /var/lib/cni
+  args:
+    creates: "/var/lib/cni"
+  failed_when: false
```

Comment:
would we need to do something similar for rkt and/or put conditionals around this for `kubelet_install_type`?